README.md

```diff
@@ -35,13 +35,17 @@ For information regarding locally cached versions of Node.js on GitHub hosted ru
 
 ### Supported version syntax
 
-The `node-version` input supports the following values:
+The `node-version` input supports the [Semantic Versioning Specification](https://github.com/npm/node-semver), for example:
 
  - Major versions: `12`, `14`, `16`
  - More specific versions: `10.15`, `14.2.0`, `16.3.0`
+ 
+And also these other syntaxes: 
```

Comment:
I do not think this remark is needed. Could you please remove it ?

Reply:
Why do you think it is not needed? The other syntax is not part of the semver spec. It should be somewhere clear?

Reply:
The section is about supported syntax for setup-node. I think it will be enough to put note about SemVer notation.